### PR TITLE
Some minor stuff

### PR DESCRIPTION
**Pull request by glenjamin**

### CODE REVIEW

File: test/routes/changePassword.spec.js [hunk 1]
```diff
@@ -1,12 +1,10 @@
 const request = require('supertest');
-const jsdom = require('jsdom');
+const cheerio = require('cheerio');
```
Comment (romidane) [inline test/routes/changePassword.spec.js]: Almost forgot about cheerio!

File: test/middleware/authentication.spec.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 const request = require('supertest');
-const express = require('express');
 
+const { setupBasicApp } = require('../test-helpers');
```
Comment (romidane) [inline test/middleware/authentication.spec.js]: ta!

File: package.json [hunk 3]
```diff
@@ -7,7 +7,7 @@
     "build": "npm run css-build",
     "css-build": "./bin/build-css",
     "lint": "eslint . --cache --max-warnings 0",
-    "test": "NODE_ENV=test mocha test/*.spec.js test/**/*.spec.js",
+    "test": "mocha",
```
Comment (romidane) [inline package.json]: won't this run on files that don't need to be tested?

Comment (glenjamin) [reply package.json]: Good question, I had to look this up to check: https://mochajs.org/#the-test-directory

Short answer is: yes
Longer answer is: but not in a way that matters

It'll load every `.js` file in `test/`, but those are either tests or helpers, so it should be fine.

**Pull request merged**